guides/common/modules/con_installing-and-configuring-insights-iop.adoc

```diff
@@ -0,0 +1,13 @@
+:_mod-docs-content-type: CONCEPT
+
+[id="installing-and-configuring-{insights-iop-id}"]
+= Installing and configuring {insights-iop}
+
+{insights-iop} analyzes system health and configuration by applying predefined rules to a small set of local data, such as installed packages, running services, and configuration settings.
+When you install {insights-iop} locally, you can generate {Insights} recommendations without sending system data to Red{nbsp}Hat services.
+
+[IMPORTANT]
+====
+With {insights-iop} enabled, you cannot use the hosted {Insights} services for hosts registered to your {Project}.
+Enabling {insights-iop} prevents you from using any Red{nbsp}Hat Hybrid Cloud Console services on the registered hosts.
```

Comment:
```suggestion
Enabling {insights-iop} prevents you from using any {RHCloud} services on hosts registered to {Project}.
```
We have an attribute for the RH HCC - please double check this is the correct attribute name.
And the hosts can be registered to HCC as well AFAIK, so let's be a bit clearer.